e2e/basic-example/basic-example.js

```diff
@@ -33,7 +33,24 @@ describe('the Basic example page', function () {
 
   describe('the dragging and dropping of nodes', function () {
     it('should allow moving a node below another node', function () {
-      // TODO
+      var handles = basicExamplePage
```

Comment:
I suggest adding a method `firstTreeItemHandler()` method to the page-object (PO) since you're re-using after moving the node. The logic to find the first node is irrelevant for the test, so moving it to the PO makes the test less verbose and thus easier to read.